### DISCUSSION TITLE
Does yolov5 take care of things outside of bounding boxes?

### DISCUSSION THREAD

**Discussion by ghost** [ultralytics/yolov5#8984] [Q&A] [answered]
Hello, here is an example:

![Untitled](https://user-images.githubusercontent.com/91971342/184851234-3923ea4a-61d8-49d6-8cc9-1fcedd3ce870.jpg)

Would it work that yolov5 only detects circles that are inside of an rectangle with a triangle on top? Because I want the program only to label this circle. If it were possible, would it be also possible to label only one pixel in a smartphone but the program still detects only smartphones?

**Comment by glenn-jocher** [comment, marked as answer]
@fennecfly yes it can detect only the examples you've labelled above. It creates detections on the object and it's surroundings.
